services/QuillLMS/spec/rails_helper.rb

```diff
@@ -85,6 +85,7 @@
   config.filter_run focus: true
   config.filter_run_excluding benchmarking: true
   config.filter_run_excluding external_api: true
+  config.filter_run_excluding big_query_snapshot: true
```

Comment:
Whoops.  Good catch on this.  I guess we've been running these on lms_rails_build too.